crypto/bio/b_addr.c

```diff
@@ -213,12 +213,10 @@ static int addr_strings(const BIO_ADDR *ap, int numeric,
             if (ret == EAI_SYSTEM) {
                 ERR_raise_data(ERR_LIB_SYS, get_last_socket_error(),
                                "calling getnameinfo()");
-                BIOerr(BIO_F_ADDR_STRINGS, ERR_R_SYS_LIB);
             } else
 # endif
             {
-                BIOerr(BIO_F_ADDR_STRINGS, ERR_R_SYS_LIB);
-                ERR_add_error_data(1, gai_strerror(ret));
+                ERR_raise_data(ERR_LIB_BIO, ERR_R_SYS_LIB, gai_strerror(ret));
```

Comment:
Appveyor is complaining about a WCHAR * to char* conversion here..

Reply:
???? How come it didn't before?

Reply:
Probably because of ...
void ERR_add_error_data(int num, ...);

Reply:
Oh, right, with ERR_add_error_data(), it was among the stdarg stuff...  so we probably treated a WCHAR* as a char*, just more silently.

I just looked at MSDN docs, and apparently, we have to use get_strerrorA() with Windows.
*Thank you Microsoft*

Reply:
Fixed, I hope